Apromore-Core-Components/Apromore-Manager/src/main/java/org/apromore/service/impl/ProcessServiceImpl.java

```diff
@@ -354,9 +354,9 @@ public ExportFormatResultType exportProcess(final String name, final Integer pro
             org.springframework.security.core.Authentication auth =
                 org.springframework.security.core.context.SecurityContextHolder.getContext().getAuthentication();
             if (auth != null) {
-                LOGGER.info("Authentication principal=" + auth.getPrincipal() + " details=" + auth.getDetails() + " thread=" + Thread.currentThread());
+                LOGGER.debug("Authentication principal={} details={} thread={}", auth.getPrincipal(), auth.getDetails(), Thread.currentThread());
             } else {
-                LOGGER.info("Authentication is null");
+                LOGGER.debug("Authentication is null");
```

Comment:
This must be LOGGER.error, not debug because null authentication is an exceptional condition (a type of error) and it must be highlighted in the log.

Reply:
If you look at the code block, it's explicitly commented as "Debug tracing of the authenticated principal".  The value `auth` is not used anywhere in the code, other than to be logged.  It's probably just a leftover from someone (likely Past Simon) trying to figure out whether the authenticated principle was accessible from that part of the code, before we had UserSessionManager to do that.

Would it be satisfactory to remove this logging message entirely?

Reply:
Removed.